tesseract_motion_planners/descartes/include/tesseract_motion_planners/descartes/descartes_collision_edge_evaluator.h

```diff
@@ -55,8 +55,6 @@ class DescartesCollisionEdgeEvaluator : public descartes_light::EdgeEvaluator<Fl
 protected:
   /** @brief The tesseract state solver */
   tesseract_kinematics::JointGroup::ConstPtr manip_;
-  /** @brief The allowed collision matrix */
-  tesseract_common::AllowedCollisionMatrix acm_;
   /** @brief A vector of active link names */
   std::vector<std::string> active_link_names_;
```

Comment:
I think the active_link_names might be something good to add to the config structure later.